### ISSUE TITLE
Installation on Windows 7 32 bits

### ISSUE THREAD

**Issue by KnightInsight** [ultralytics/yolov5#13273]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hi, may I know is yolov5 able to install on windows 7 32 bits?

**Comment by glenn-jocher** [comment]
@KnightInsight yOLOv5 requires a 64-bit operating system. Windows 7 32-bit is not supported. For best results, please use a 64-bit OS.

**Comment by KnightInsight** [comment]
How about if I just want to load onnx model for detection? Is the library support?

**Comment by glenn-jocher** [comment]
@KnightInsight yes, YOLOv5 supports ONNX model inference. You can use the `export.py` script to convert your model to ONNX and then load it for detection.
